src/middleware/session/state/cookie.lisp

```diff
@@ -15,16 +15,20 @@
            :generate-sid
            :extract-sid
            :expire-state
-           :finalize-session))
+           :finalize-session
+           :samesite-type))
 (in-package :lack.middleware.session.state.cookie)
 
+(deftype samesite-type () `(member lax strict none))
```

Comment:
It might be preferable to use keywords so that users don't need to import these symbols.

Reply:
I understood that point, therefore I close commit once.